src/App/Components/DisplayArticle/articleBody.js

```diff
@@ -13,7 +13,7 @@ const ArticleBody = (props) => {
   } = props;
   const formattedDate = moment(createdAt).format('Do MMM YYYY');
   return (
-    <div className="col-xl-9 col-lg-9 col-md-12 col-sm-12 col-12">
+    <div className="col-xl-12 col-lg-9 col-md-12 col-sm-12 col-12">
```

Comment:
had to adjust the design of the read article, that is why I have some changes in this files